modules/account-lib/src/coin/eth/utils.ts

```diff
@@ -154,6 +156,16 @@ export function flushTokensData(forwarderAddress, tokenAddress): string {
   return addHexPrefix(Buffer.concat([method, args]).toString('hex'));
 }
 
+/**
+ * Get the data required to make a flush native coins contract call
+ */
+export function flushCoinsData(): string {
+  const params = [];
+  const method = EthereumAbi.methodID('flush', flushCoinsTypes);
+  const args = EthereumAbi.rawEncode(flushCoinsTypes, params);

```

Comment:
We could probably just make this an empty array and skip the types / params right?

Reply:
I.e. I think this function can just return `addHexPrefix(method).toString('hex')` for the same result

Reply:
Yeah, just wanted to make it a bit more formalized :P

Reply:
Totally fine then!